### PR TITLE
feat/1251: Add Namada dry run config, feature flags

### PR DESCRIPTION
**Pull request by jurevans**
Partially resolves #1251

- Add schema for `features`
- Update current chains to specify default-enabled features
- Add configs for `namadadryrun`
- Updated `slip-044` for NAM (118 is ATOM, 877 is assigned to Nam) See https://github.com/satoshilabs/slips/blob/master/slip-0044.md

### CODE REVIEW

File: namadadryrun/chain.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  "network_type": "testnet",
+  "website": "https://namada.net",
+  "pretty_name": "Namada",
+  "chain_type": "namada",
+  "chain_id": "namada-dryrun.abaaeaf7b78cb3ac",
+  "slip44": 118,
+  "bech32_prefix": "tnam",
+  "daemon_name": "unam",
+  "key_algos": ["ed25519"],
+  "features": [],
```
Comment (jurevans) [inline namadadryrun/chain.json:14]: Empty features indicates `Phase 1` features

File: namadadryrun/chain.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "namadahousefire",
```
Comment (emccorson) [inline namadadryrun/chain.json:3]: I haven't checked the PR fully, but I think `chain_name` should be `namadadryrun` here and in the assetlist.json.

Comment (jurevans) [reply namadadryrun/chain.json:3]: ah ok thanks! I completely missed that

Comment (jurevans) [reply namadadryrun/chain.json]: fixed!

**Review by euharrison — APPROVED** [review]

**Pull request merged**